### PR TITLE
[skip ci] Updated INSTALL.md.

### PR DESCRIPTION
**Pull request by AmyYH**
Signed-off-by: Amy Hong <yun.hong@zilliz.com>

**What type of PR is this?**

- [ ] API-change
- [ ] BUG
- [ ] Improvement
- [x] Documentation
- [ ] Feature
- [ ] Test and CI

**Which branch you want to cherry-pick to?**

Not Available

**Which issue(s) this PR fixes:**

Fixes #

**What this PR does / why we need it:**

Not Available

**Special notes for your reviewer:**

Not Available

**Additional documentation (e.g. design docs, usage docs, etc.):**

Not Available

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -58,49 +58,36 @@ For GPU-enabled version, you will also need:
...
-$ cd [Milvus root path]/core
-$ ./ubuntu_build_deps.sh
-```
-
-##### Install in CentOS
-
-```shell
-$ cd [Milvus root path]/core
-$ ./centos7_build_deps.sh
+$ cd /home
```
Comment (JinHai-CN) [inline INSTALL.md:66]: remove this line

File: INSTALL.md [hunk 2]
```diff
@@ -58,49 +58,36 @@ For GPU-enabled version, you will also need:
...
 
-If you want to know the complete build options, run the following command.
+#### Step 3 Build Milvus source code
 
 ```shell
-$./build.sh -h
+$ ./build.sh -t Debug
 ```
 
-When the build is completed, everything that you need in order to run Milvus will be installed under `[Milvus root path]/core/milvus`.
```
Comment (JinHai-CN) [inline INSTALL.md:102]: Why remove this sentence?

File: INSTALL.md (hunk 1 above)
Comment (AmyYH) [reply INSTALL.md]: Removed.

File: INSTALL.md (hunk 2 above)
Comment (AmyYH) [reply INSTALL.md:102]: Added back now.

**Pull request closed without merging**